### ISSUE TITLE
Error during migrate:generate of custom data type in postgres 13. 

### ISSUE THREAD

**Issue by nzsakib** [kitloong/laravel-migrations-generator#75]
Ran this command after installing the package: 
```php
php artisan migrate:generate -c pgsql
```
We have custom data type for some columns in postgres 13. 
example: 
```sql
create type phone_number_type as enum ('TOLL-FREE', 'SHORTCODE', '10 DLC', 'ALPHANUMERIC', 'TOLLFREE');
```
Then used in table like this: 
```sql
create table phone_numbers
(
    id          bigserial
        constraint phone_numbers_pk primary key,
    phone       varchar(60),
    type        phone_number_type,
);
```
This package gives following error during migration generate: 
![image](https://user-images.githubusercontent.com/8240768/151647469-25a734d3-e429-4e09-8d07-3bbed643e94d.png)

**Comment by kitloong** [comment]
Duplicate of #65

**Comment by kitloong** [comment]
Released with tag `v6.4.0`
